cmd/commento/main.go

```diff
@@ -17,6 +17,10 @@ func main() {
 		Die(err)
 	}
 	
+	if err := godotenv.Load(); err != nil {
```

Comment:
Ok, checkout the latest commit (why didn't it add the changes here?) to see the updates.